xmtp_mls/src/subscriptions/process_message/factory.rs

```diff
@@ -107,6 +108,7 @@ where
             self.0.clone(),
             msg.group_id.clone(),
             None,
+            ConversationType::Group,
```

Comment:
This is the one place the new conversation type needs to be added where we dont know its value. I think we're also populating dummy data for dm id here and the code path looks like conversation_type is not used, so I think this is safe for now, but seems like a code smell.